integration_tests/configs/ibc.jsonnet

```diff
@@ -2,6 +2,8 @@ local config = import 'default.jsonnet';
 
 config {
   'cronos_777-1'+: {
+    cmd: 'cronosd',
+    'start-flags': '--trace --log_level info',
```

Comment:
```suggestion
```

this seems to be the default config, no need to override.

Reply:
yeah I added because its easier if we want to change the log level for debugging